src/middleware.ts

```diff
@@ -3,6 +3,7 @@ import { NextRequest, NextResponse } from 'next/server';
 
 const FREE_ACCESS_PAGES = [
   '/',
+  '/home',
```

Comment:
@tolokoban a small question: what's the reason to have `/home` when the root page is already serving the same content?